sklearn/tests/test_grid_search.py

```diff
@@ -220,19 +220,10 @@ def test_grid_search_score_method():
     # Check warning only occurs in situation where behavior changed:
```

Comment:
@jnothman : I think that the code is good now.

@Sentient07 : maybe you can change the comment line to describe more accurately the test.